pytest_mozwebqa/selenium_client.py

```diff
@@ -139,6 +142,16 @@ def start_client(self):
             capabilities.update(webdriver.DesiredCapabilities.OPERA)
             self.selenium = webdriver.Opera(executable_path=self.opera_path,
                                             desired_capabilities=capabilities)
+        elif self.driver.upper() == 'BROWSERSTACK':
```

Comment:
This is for consistency with the other driver string comparisons in the file. I've no real preference between upper or lower. Out of curiosity, why do say upper is less pythonic?